# i18n-proposal: ./src/creatures/npcs/npcs.cpp
# Generated: 2026-02-15T19:30:44.127327Z
# Category: server
# Transforms: 2

--- ./src/creatures/npcs/npcs.cpp:37
-		g_logger().warn("[NpcType::loadCallback] - Event not found");
+		g_logger().warn(i18n::g_translator().get("misc.npcs.npctype_loadcallback_event", player->getLocale()));
# key: misc.npcs.npctype_loadcallback_event
# en: [NpcType::loadCallback] - Event not found

--- ./src/creatures/npcs/npcs.cpp:111
-		return g_luaEnvironment().loadFile(coreFolder + "/npclib/load.lua", "load.lua") == 0;
+		return g_luaEnvironment().loadFile(coreFolder + "/npclib/load.lua", i18n::g_translator().get("misc.npcs.load_lua", player->getLocale())) == 0;
# key: misc.npcs.load_lua
# en: load.lua

